arduino-ide-extension/src/browser/settings.tsx

```diff
@@ -681,12 +693,17 @@ export class SettingsWidget extends ReactWidget {
     @inject(WindowService)
     protected readonly windowService: WindowService;
 
+    @inject(CommandRegistry)
```

Comment:
FYI: There is the `CommandService` API, which's sufficient for this case, as you want to only execute a command. No need to change, just wanted to share this with you.

Reply:
👍 I'm gonna check the difference between the `CommandService` and the `CommandRegistry`